pkg/components/exporter/exporter.go

```diff
@@ -0,0 +1,341 @@
+						Effect: corev1.TaintEffectNoSchedule, Key: "node-role.kubernetes.io/master",
+					}},
+					Containers: []corev1.Container{{
+						Name:            "kepler-exporter",
+						SecurityContext: &corev1.SecurityContext{Privileged: pointer.Bool(true)},
+						Image:           exporter.Image,
+						Command: []string{
+							"/usr/bin/kepler",
+							"-address", bindAddress,
+							"-enable-gpu=true",
+							"-enable-cgroup-id=true",
+							"-v=1",
+							"-kernel-source-dir=/usr/share/kepler/kernel_sources",
+						},
+						Ports: []corev1.ContainerPort{{
+							ContainerPort: int32(exporter.Port),
+							Name:          "http",
+						}},
+						LivenessProbe: &corev1.Probe{
+							ProbeHandler: corev1.ProbeHandler{
+								HTTPGet: &corev1.HTTPGetAction{
+									Path:   "/healthz",
+									Port:   intstr.IntOrString{Type: intstr.Int, IntVal: int32(exporter.Port)},
+									Scheme: "HTTP",
+								},
+							},
+							FailureThreshold:    5,
+							InitialDelaySeconds: 10,
+							PeriodSeconds:       60,
+							SuccessThreshold:    1,
+							TimeoutSeconds:      10},
+						Env: []corev1.EnvVar{
+							{Name: "NODE_IP", ValueFrom: k8s.EnvFromField("status.hostIP")},
+							{Name: "NODE_NAME", ValueFrom: k8s.EnvFromField("spec.nodeName")}},
+						VolumeMounts: []corev1.VolumeMount{
+							{Name: "lib-modules", MountPath: "/lib/modules"},
+							{Name: "tracing", MountPath: "/sys"},
+							{Name: "kernel-src", MountPath: "/usr/src/kernels"},
+							{Name: "kernel-debug", MountPath: "/sys/kernel/debug"},
+							{Name: "proc", MountPath: "/proc"},
+							{Name: "cfm", MountPath: "/etc/kepler/kepler.config"},
+						}, // VolumeMounts
+					}}, // Container: kepler /  Containers
+					Volumes: []corev1.Volume{
+						k8s.VolumeFromHost("lib-modules", "/lib/modules"),
+						k8s.VolumeFromHost("tracing", "/sys"),
+						k8s.VolumeFromHost("proc", "/proc"),
+						k8s.VolumeFromHost("kernel-debug", "/sys/kernel/debug"),
+						k8s.VolumeFromHost("kernel-src", "/usr/src/kernels"),
+						k8s.VolumeFromConfigMap("cfm", ConfigmapName),
+					}, // Volumes
+				}, // PodSpec
+			}, // PodTemplateSpec
+		}, // Spec
+	}
+
+}
+
+func NewConfigMap(d components.Detail, k *v1alpha1.Kepler) *corev1.ConfigMap {
+	if d == components.Metadata {
+		return &corev1.ConfigMap{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: corev1.SchemeGroupVersion.String(),
+				Kind:       "ConfigMap",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      ConfigmapName,
+				Namespace: components.Namespace,
+				Labels:    labels,
+			},
+		}
+	}
+
+	exporter := k.Spec.Exporter
+	bindAddress := "0.0.0.0:" + strconv.Itoa(exporter.Port)
+
+	return &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: corev1.SchemeGroupVersion.String(),
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      ConfigmapName,
+			Namespace: components.Namespace,
+			Labels:    labels,
+		},
+		Data: map[string]string{
```

Comment:
@husky-parul has made some changes to the configmap recently, and some defaults are also updated in [kepler repo](https://github.com/sustainable-computing-io/kepler/blob/main/manifests/config/exporter/exporter.yaml#L16)

Reply:
@rootfs shall I address this is separate PR?  - filed 
https://github.com/sustainable-computing-io/kepler-operator/issues/111